### DISCUSSION TITLE
Always open leaderboard

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#29183] [Ideas]
I was looking for similar discussions, but I couldn't find any. I would like to be able to always see the leaderboard open during the game, and not only when holding down Ctrl or during breaks. It's very inconvenient.

Thank you.

**Comment by SupDos** [comment]
Previously discussed in https://github.com/ppy/osu/discussions/22040, and was PR'd in https://github.com/ppy/osu/pull/24947 and https://github.com/ppy/osu/pull/27285, but the devs wanted to wait until the leaderboard is skinnable to add this setting to the skin editor.

Since it's been brought up, I'll add my comment as well - I'd like having this as an option in settings instead of the skin editor so that it can be toggled with a keybind (I don't think skin editor options can be binded to a key).
This would allow being able to press tab or whatever key you choose to switch between the full size leaderboard and the small size one, instead of hiding it completely. At the moment you can sort of achieve this by holding control (as OP says), but having it in settings would allow it to be a toggle instead.
